docs/topics/native/native-objc-interop.md

```diff
@@ -208,6 +210,17 @@ Objective-C does not support packages in a framework. If the Kotlin compiler fin
 which have the same name but different packages, it renames them. This algorithm is not stable yet and can change between
 Kotlin releases. To work around this, you can rename the conflicting Kotlin classes in the framework.
 
+#### Strong linking
+
+Whenever you use an Objective-C class in the Kotlin source, it's marked as a strongly linked symbol. The resulting build
+artifact mentions related symbols as strong external references.
+
+This means that the app tries to link symbols during the launch dynamically, and if they are unavailable, the app crashes.
+Symbols might not be available on a particular device or OS version. It happens even if symbols were never used.
```

Comment:
I would rewrite it as follows:
The crash happens even if symbols were never used. Symbols might be unavailable on a particular device or OS version.

This seems a bit less ambiguous to me.

Reply:
thank you! that works